python-main.js

```diff
@@ -342,11 +349,6 @@ function web_editor(config) {
             EDITOR.setCode(config.translate.code.start);
         }
         EDITOR.ACE.gotoLine(EDITOR.ACE.session.getLength());
-        // If configured as experimental update editor background to indicate it
-        if(config.flags.experimental) {
-            EDITOR.ACE.renderer.scroller.style.backgroundImage = "url('static/img/experimental.png')";
-            $("#known-issues").removeClass('hidden');
-        }
```

Comment:
Also not sprightly necessary for this PR, but moved this feature flag config to the `setupFeatureFlags()` functions, where other features are configured.